hs_file_types/models/reftimeseries.py

```diff
@@ -709,21 +817,25 @@ def get_copy(self):
 
 def _extract_metadata(resource, logical_file):
     # add resource level title if necessary
-    if resource.metadata.title.value == 'Untitled resource':
+    if resource.metadata.title.value.lower() == 'untitled resource' \
+            and logical_file.metadata.has_title_in_json:
         resource.metadata.update_element('title', resource.metadata.title.id,
-                                         value=logical_file.metadata.title)
+                                         value=logical_file.dataset_name)
 
-    # add resource level abstract id necessary
-    if resource.metadata.description is None:
-        resource.metadata.create_element('description', abstract=logical_file.metadata.abstract)
+    # add resource level abstract if necessary
+    if resource.metadata.description is None and logical_file.metadata.has_abstract_in_json:
+        resource.metadata.create_element('description',
+                                         abstract=logical_file.metadata.get_abstract_from_json())
```

Comment:
@pkdash If eliminating ```has_abstract_in_json()``` but using ```get_abstract_from_json()``` instead, the load json dict will only need to be called once, and I think it is simpler also.